packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Connect your SCIM API service to Okta
+excerpt: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+meta:
+  - name: description
+    content: Create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+layout: Guides
+---
+
+This guide — part of the [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integration-overview/) series — teaches you how to create a SCIM integration directly within Okta, configure the SCIM options for that integration, and check the attributes and corresponding mappings.
+
+## Getting connected
+
+After you have a SCIM implementation that passes all of the Runscope tests, as discussed in the [previous part of the series](/docs/guides/scim-provisioning-integration-prepare/main/#test-your-scim-api), you need to create your SCIM integration directly within Okta.
+
+Begin by signing up for an [Okta developer account](https://developer.okta.com/signup/).
+
+1. After you request the developer account and have received the initial email, click the link to go to your Admin Console.
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Click **Browse App Catalog**.
+1. Search for "SCIM 2.0" or "SCIM 1.1" (your choice depends on which version your SCIM server supports). You'll see three different SCIM template applications, one for each of the three authentication methods that you can use to connect to your SCIM implementation (Basic Auth, Header Auth, or OAuth Bearer Token).
+1. Click **Add** on the template that you want to use and complete the add steps.
+1. On the **General Settings** page, give your integration a descriptive name and specify whether you want it to be hidden from general and mobile users. Additionally, you can decide if you want to have your users automatically be logged in when they reach the landing page in their web browser. Click **Next**.
```

Comment:
How about ("in their web browser" is redundant as landing pages are understood to be in a web browser) :
1. On the **General Settings** page: set the name of your application, choose if it's hidden from general and mobile users, and choose if users' are automatically logged in from the landing page. Click **Next**.

Reply:
done

Reply:
Updated.